en/drafts/originals/designing-a-timeline-tabletop-simulator.md

```diff
@@ -430,99 +424,97 @@ Similar to the example that was walked through above, [`PH_nandeck_Your_Timeline
+This next section will describe how I turned my 'Windsor Timeline' deck into digital format for TTS. Then, generalized instructions will explain how to follow the same process with the game assets we just used to make our simple six-card starter deck. 
 
-### Printing Card Faces and Card Backs Separately for _Windsor Timeline_
+### Printing Card Faces and Card Backs Separately for 'Windsor Timeline'
 
-Unlike _nanDECK_ which generates each card separately, _Tabletop Simulator_ requires two image files (one for each side of a deck) to be in a uploaded in standardized format so that it can parse each image into multiple cards. A normal card deck will generally have an illustration on the card 'back' (such as a mosaic pattern), while information and symbols will appear on the card 'face' (such as the 4 of Clubs). So, in other words, TTS requires one image file that portrays all the cards' 'faces' and another image that portrays all the cards' 'backs.'
+Unlike nanDECK, which generates each card separately, Tabletop Simulator requires two image files (one for each side of the deck) to be uploaded in standardized format, so that it can parse each one into multiple cards. A normal card deck will generally have an illustration on the card back (such as a mosaic pattern), while information and symbols appear on the card face (such as the 4 of Clubs). In other words, TTS requires one image file that portrays all the card faces and another file that portrays all the card backs.
 
-So before we can upload our _Timeline_ deck into TTS, we will need to regenerate our cards into this required format. To do this, it is necessary to split our original spreadsheet into two spreadsheets. The first spreadsheet will generate the deck's 'face' and as such, will have the necessary information to generate facts, images and years. This spreadsheet will no longer need duplicate entries that missing entries in the 'years' column. The second spreadsheet will be identical to the first with columns for 'Fact' and 'Image', but will not include the 'Year.' 
+Before we can upload our _Timeline_ deck into TTS, we will need to regenerate our cards into this required format. To do this, we have to split our original spreadsheet into two. The first spreadsheet will generate the deck's faces and, as such, will have the necessary information to generate facts, images and years. This spreadsheet will no longer need the duplicate entries without the 'years'. The second spreadsheet will be identical to the first, except that it won't include the 'year' column. 
 
-Then it is necessary to change our script to reflect that the spreadsheets are going to be used to generate one of two files needed by TTS:
+We will have to change our script to reflect that the spreadsheets are going to be used to generate one of two files needed by TTS:
```

Comment:
I wasn't exactly sure what was meant here – would you be able to rephrase this comment to clarify?

Reply:
Is this clearer?

"Now that we have two spreadsheets, we need to amend our previous set of instructions in the nanDECK interface in a couple of places before we can run the script."